src/ConnectedMode.UnitTests/Binding/CSharpVbBindingConfigTests.cs

```diff
@@ -20,16 +20,14 @@
 
 using System;
 using System.IO;
-using System.Linq;
+using System.IO.Abstractions.TestingHelpers;
 using SonarLint.VisualStudio.Core.CSharpVB;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Binding.UnitTests
 {
     [TestClass]
     public class CSharpVbBindingConfigTests
     {
-        public TestContext TestContext { get; set; }
```

Comment:
No longer required. It was only needed before because a test accessed the real file system, and used `TestContext` to work out where to write a file.